src/sso/interfaces/connection.interface.ts

```diff
@@ -7,6 +7,22 @@ export interface ConnectionDomain {
 }
 
 export interface Connection {
+  object: 'connection';
+  id: string;
+  organizationId?: string;
+  name: string;
+  connectionType: ConnectionType;
```

Comment:
What do we think about renaming this to `type`?

Now that we have the deserialization layer in place, we can translate the `connection_type` field to `type`.

This would align with `Directory`s, which already use `type`.

Reply:
I'm going to take that approach! I think that feels measured.